packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -83,6 +83,8 @@ const EventIconMap: Record<EventCode, React.FC> = {
   [eventCodes.PORTFORWARD_FAILURE]: Icons.Info,
   [eventCodes.SUBSYSTEM]: Icons.Info,
   [eventCodes.SUBSYSTEM_FAILURE]: Icons.Info,
+  [eventCodes.LOCK_CREATED]: Icons.Lock,
+  [eventCodes.LOCK_DELETED]: Icons.Unlock,
```

Comment:
I'd keep this the same as `Icons.Lock`, to be consistent with others (we don't have separate icons to denote these create/delete actions for other events)